documentation/Data-Display.md

```diff
@@ -1,38 +1,34 @@
-# Outcomes Display
+# Data Display
 
 <!-- toc -->
 
 ## Contents
   - [API](#api)
-    - [`/outcomes`](#outcomes)
-    - [`/outcomes/table/<tableName>?<queries>`](#outcomestabletablenamequeries)
-      - [Query parameters](#query-parameters)
-    - [`/outcomes/table/<tableName>/item/<id>`](#outcomestabletablenameitemid)
   - [Configuration](#configuration)
-    - [`outcomes_display` table](#outcomes_display-table)
-    - [`outcome_display_column_definition` table](#outcome_display_column_definition-table)
+    - [`data_displays` table](#data_display-table)
+    - [`data_display_column_definition` table](#data_display_column_definition-table)
```

Comment:
Hmmmm a little annoyed with this name. Couldn't the endpoint be data-tables instead?
data-displays is a bit strange. I'm sure we talked about how to name this before and had a better name

Reply:
Sure, the only other name I could think of (that we may have discussed?) is "data_views". "Display"/ "View" is what it is -- it's very important that we don't call it "data tables", since this is not a list of data tables, it's a list of different ways of *viewing* (or displaying) what's _in_ the data tables.

What do you think of "data_views"?